internal/network/command_dns_record_create.go

```diff
@@ -0,0 +1,98 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingaccesspointv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking-access-point/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+const (
+	privateLinkAccessPoint = "PrivateLinkAccessPoint"
+)
+
+func (c *command) newDnsRecordCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create [name]",
+		Short: "Create a DNS record.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE:  c.dnsRecordCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Create a DNS record.",
+				Code: "confluent network dns record create --gateway gw-123456 --access-point ap-123456 --fqdn www.example.com",
```

Comment:
We have `--fqdn` in the example and `--fully-qualified-domain-name` below when we declare the flag. Do we have an alias somewhere? Not sure if i missed it...

Reply:
I just forgot to change the flag in the example.

Reply:
Can we shorten this flag in any way? How about `--domain-name`?

Reply:
That would simplify it. What do you think, @syoung-cflt ?

"Fully qualified" would still be in the flag descriptions.

Reply:
@kabraham13 The is for creating a DNS Record. Is it fine if we use `--domain-name` instead of `--fully-qualified-domain-name` for the `fqdn` attribute?

Reply:
Lets just use `--domain`

Reply:
Any reason we don't use that in the API as well? @syoung-cflt @ejagade

Reply:
@syoung-cflt if the LoE is small, let's change it to `domain` . Using `domain` is more consistent with rest of the attribute names used for DNS
we also should let UI folks know about this change.

Reply:
updated the sdk - https://github.com/confluentinc/ccloud-sdk-go-v2-internal/releases/tag/networking-access-point%2Fv0.0.2
will update networking service now